### PR TITLE
fix(q): Make copy/text changes to Feature Development capability

### PR DESCRIPTION
**Pull request by sarayev**
## Problem
Needed to make visual text&copy changes to Feature Development capability. 

## Solution
Making visual, copy/text changes by
 - Removing `Application Development` menu group
 - Updating command description for `/dev` command
 - Updating welcome message for `/dev` command
 - Updating followup message of `/dev` command

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .changes/next-release/Bug Fix-4480804e-ed2b-4d53-98f6-c22ae10fdbf0.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Amazon Q Code Feature Development: Remove Application Development menu group, update welcome message and menu item description for /dev command"
```
Comment (jpinkney-aws) [inline .changes/next-release/Bug Fix-4480804e-ed2b-4d53-98f6-c22ae10fdbf0.json:3]: You can probably drop the Code for "Amazon Q feature development"

**Comment by jpinkney-aws** [comment]
It looks like the tests are failing because of this change

File: packages/core/src/amazonq/webview/ui/quickActions/generator.ts [hunk 2]
```diff
@@ -25,13 +25,12 @@ export class QuickActionGenerator {
...
                 ? [
                       {
-                          groupName: 'Application Development',
                           commands: [
                               {
                                   command: '/dev',
                                   placeholder: 'Briefly describe a task or issue',
                                   description:
-                                      'Use all project files as context for code suggestions (increases latency).',
+                                      'Plan and implement new functionality across multiple files in your workspace.',
```
Comment (dogusata) [inline packages/core/src/amazonq/webview/ui/quickActions/generator.ts:33]: We also have a [**grouping change**](https://github.com/aws/aws-toolkit-vscode/pull/4800) for the quick action commands list as requested from the UX team.

Comment (sarayev) [reply packages/core/src/amazonq/webview/ui/quickActions/generator.ts:33]: Yes, seems that some changes overlap. I'm still waiting for a small fix for one of the texts in my PR. Can rebase mine on top of yours once you merge.

**Review by kumsmrit — APPROVED** [review]

**Review by vishaalmehrishi — APPROVED** [review]

**Review by jpinkney-aws — APPROVED** [review]

**Pull request merged**